blog/forms.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from .models import Commentary
```

Comment:
Use absolute imports only and separate different groups of imports with blank line. check other files as well for this issue